python/ray/tune/execution/checkpoint_manager.py

```diff
@@ -100,10 +106,12 @@ def newest_persistent_checkpoint(self):
     @property
     def newest_checkpoint(self):
         """Returns the newest checkpoint (based on training iteration)."""
```

Comment:
Any suggestions how to account for the fact that we are sometimes not returning the "newest checkpoint" anymore? Might be confusing, since forced checkpoints always get preferred even if they are staler.